docs/non-functional-requirements/privacy/README.md

```diff
@@ -0,0 +1,6 @@
+# Privacy fundamentals
+
+Private data handling and protection requires both the proper design of software,
+systems and databases, as well as the implementation of organizational processes and procedures.
+
+In general, developers working on [ISE](../ISE.md) projects should adhere to Microsoft's recommended standard practices and regulations on Privacy and Data Handling.
```

Comment:
should this link to the other files in the directory? 
this seems to have been the case but removed?

Reply:
There is one to it in the docs/design/design-patterns/non-functional-requirements-capture-guide.md